modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/internal/TcpDiscoveryNode.java

```diff
@@ -499,6 +499,13 @@ public void clientAliveTime(long aliveTime) {
         this.aliveCheckTime = U.currentTimeMillis() + aliveTime;
     }
 
+    /**
+     * @return Client alive check time.
+     */
+    public long getAliveCheckTime() {
```

Comment:
By CC this method should be called aliveCheckTime()

Reply:
You should probably call it clientAliveTime() to match setter

Reply:
Agree. Sorry, forgot about this rule.